stackit/internal/services/redis/credential/resource.go

```diff
@@ -103,6 +103,7 @@ func (r *credentialResource) Schema(_ context.Context, _ resource.SchemaRequest,
 		"credential_id": "The credential's ID.",
 		"instance_id":   "ID of the Redis instance.",
 		"project_id":    "STACKIT Project ID to which the instance is associated.",
+		"uri":           "Connection uri. Currently the returned uri doesn't support ssl, but you can enable it by using rediss:// instead of redis://. We will provide the ssl connection in a future release",
```

Comment:
```suggestion
		"uri":           "Connection URI. Currently the returned uri doesn't support SSL, but you can enable it by using `rediss://` instead of `redis://`. We will provide the SSL connection in a future release",
```

Reply:
The `datasource` file also has this